api/v1beta1/cinderapi_types.go

```diff
@@ -28,6 +28,8 @@ type CinderAPITemplate struct {
 
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=1
+	// +kubebuilder:validation:Maximum=32
```

Comment:
Why so many? Tripleo generally deploys 3 (one per controller).

Reply:
This follows the value commonly used in the other operators.

Also, because now we do not we provide a way to configure wsgi processes/threads, it's more likely that users need to scale out api pods. So having a relatively high maximum makes sense, IMO.